ament_uncrustify/ament_uncrustify/main.py

```diff
@@ -65,7 +65,7 @@ def main(argv=sys.argv[1:]):
         help='Exclude specific file names and directory names from the check')
     parser.add_argument(
         '--language',
-        choices=['C', 'C++'],
+        choices=['C', 'CPP'],
```

Comment:
I don't think the `C++` here is meant for `uncrustify`. I think it's meant for `ament_uncrustify`, and making this change will break the dictionary [here](https://github.com/ament/ament_lint/pull/272/files#diff-6e94bb1449472e0ebabe73155ff68bf9546cda13785946c5891a8afb741b9e04R108).

Can you double check?

Reply:
@mm318, can you confirm the link to the dictionary?

Reply:
Nevermind, I see what you meant. 

I think you're right that it breaks the dictionary here.
https://github.com/ament/ament_lint/blob/d17140b716eb11f7cc8b3d686104ed2167ce9818/ament_uncrustify/ament_uncrustify/main.py#L107-L109

@MiguelCompany, the dictionary could be updated to use `CPP` and thus make `ament_uncrustify` more consistent with `uncrustify`. I believe line 108 is the only one that would need to change.

Reply:
Nevermind.